spec/support/forgot_password_page.rb

```diff
@@ -0,0 +1,7 @@
+class ForgotPasswordPage < SitePrism::Page
+  set_url '/users/password/new'
+  set_url_matcher %r{/users/password/new}
+
+  element :email_field, '#user_email'
```

Comment:
Yeah `t-` classes are preferable as it's less likely they'll be used for anything other than the test hook. IDs could be relied on elsewhere (probably js) and things can get messy quite quickly